### PR TITLE
Conformance

### PR DESCRIPTION
**Pull request by qinqon**

### CODE REVIEW

**Comment by danielBelenky** [comment]
Why you chose https://github.com/vmware-tanzu/sonobuoy/ over https://github.com/cncf/k8s-conformance?

**Comment by rmohr** [comment]
> Why you chose https://github.com/vmware-tanzu/sonobuoy/ over https://github.com/cncf/k8s-conformance?

That's what k8s-conformance refers to.

File: cluster-provision/k8s/1.16/conformance.json [hunk 1]
```diff
@@ -0,0 +1 @@
...ets","deployments","endpoints","ingresses","jobs","leases","limitranges","mutatingwebhookconfigurations","namespaces","networkpolicies","nodes","persistentvolumeclaims","persistentvolumes","poddisruptionbudgets","pods","podlogs","podsecuritypolicies","podtemplates","priorityclasses","replicasets","replicationcontrollers","resourcequotas","rolebindings","roles","servergroups","serverversion","serviceaccounts","services","statefulsets","storageclasses","validatingwebhookconfigurations","volumeattachments"],"Filters":{"Namespaces":".*","LabelSelector":""},"Limits":{"PodLogs":{"Namespaces":"","SonobuoyNamespace":true,"FieldSelectors":[],"LabelSelector":"","Previous":false,"SinceSeconds":null,"SinceTime":null,"Timestamps":false,"TailLines":null,"LimitBytes":null,"LimitSize":"","LimitTime":""}},"QPS":30,"Burst":50,"Server":{"bindaddress":"0.0.0.0","bindport":8080,"advertiseaddress":"","timeoutseconds":10800},"Plugins":[{"name":"e2e"},{"name":"systemd-logs"}],"PluginSearchPath":["./plugins.d","/etc/sonobuoy/plugins.d","~/sonobuoy/plugins.d"],"Namespace":"sonobuoy","WorkerImage":"sonobuoy/sonobuoy:v0.18.2","ImagePullPolicy":"IfNotPresent","ImagePullSecrets":"","ProgressUpdatesPort":"8099"}
```
Comment (rmohr) [inline cluster-provision/k8s/1.16/conformance.json:1]: Maybe you want to format them.

**Pull request merged**